ycmd/completers/general/tests/filename_completer_test.py

```diff
@@ -233,3 +236,80 @@ def EnvVar_Undefined_Garbage_3_test( self ):
 
     os.environ.pop( 'YCM_TEST_td' )
     eq_( [ ], data )
+
+
+def WorkingDir_Use_File_Path_test( ):
+
+  assert os.getcwd() != DATA_DIR, ( "Please run this test from a different "
+                                    "directory" )
+
+  options = user_options_store.DefaultOptions()
+  options.update( {
+    'filepath_completion_use_working_dir': 0
+  } )
+  completer = FilenameCompleter( options )
+
+  data = sorted( _CompletionResultsForLine( completer, 'ls ./include/' ) )
+  eq_( [
+        ( 'Qt',       '[Dir]' ),
+        ( 'QtGui',    '[Dir]' ),
+      ], data )
+
```

Comment:
again, sorry I'm not sure where you're expecting another blank line. perhaps there's some style guideline i'm not aware of?!